### DISCUSSION TITLE
Can we start multiple instance of tdlight in one java app

### DISCUSSION THREAD

**Discussion by gillbates** [tdlight-team/tdlight-java#73] [Q&A] [answered]
Hi team,

thanks first for all the support along to bring tdlib into java.
I googled and found there is workaround in tdlib to support multiple accounts.
I wonder if we could also do this with tdlight?

cheers,
gill

**Comment by cavallium** [comment, marked as answer]
Just create multiple clients using ClientManager, as in the example.
Every TelegramClient is completely separated from each other
